frontend/src/layouts/content-layout.vue

```diff
@@ -167,7 +167,7 @@ export default defineComponent({
   width: var(--filter-sidebar-width);
 }
 .app {
-  grid-template-areas: "header header" "main main";
+  grid-template-areas: "header header" "main main" "global-audio global-audio";
```

Comment:
Since the grid only has one column when `isSidebarVisible` is `false` (see L10), this can be simplified.

```suggestion
  grid-template-areas: "header" "main" "global-audio";
```

Reply:
Nice, I haven't thought about that 🤦 Thanks!